crypto/whrlpool/wp_dgst.c

```diff
@@ -174,7 +174,7 @@ void WHIRLPOOL_BitUpdate(WHIRLPOOL_CTX *c, const void *_inp, size_t bits)
                 goto reconsider;
             } else
 #endif
-            if (bits >= 8) {
+            if (bits > 8) {
```

Comment:
There is comment in corresponding `else`, which would be appropriate to adjust. As "remaining less than or 8 bits".